### PR TITLE
Wait for sentiment build

### PR DESCRIPTION
**Pull request by Tahnan**

### CODE REVIEW

File: tests/test_v5_client.py [hunk 1]
```diff
@@ -103,3 +105,134 @@ def test_failing_requests(requests_mock):
...
+
+    # If there is no build: error
+    requests_mock.get(project_url, json={'last_build_info': {}})
+    with pytest.raises(ValueError, match='not building'):
+        client.wait_for_build()
+
+    # If the build succeeds: the project's last build info
+    requests_mock.get(
+        project_url,
+        _last_build_infos_to_mock_returns(
```
Comment (slibs63) [inline tests/test_v5_client.py:144]: I'm confused about why we pass this.

Comment (Tahnan) [reply tests/test_v5_client.py:144]: requests-mock lets you specify a list of returns to use in sequential calls.  Lines 142-147 are saying "Tell the mock request that the responses to a `get` call to `project_url` are the following list....".  It's directly analogous to line 137 above, which says "the response to a `get` call to `project_url` is a JSON-encoded `{'last_build_info': {}}`".

Comment (slibs63) [reply tests/test_v5_client.py:144]: Oh, I see -- `client.wait_for_build()` is hitting that endpoint multiple times.  I wasn't seeing where we were calling it more than once.

**Comment by slibs63** [comment]
Okay, I think I buy all of this.  Now comes the question about whether this is safe to merge as long as we don't PyPI it.  Was a conclusion ever reached on that?

**Comment by slibs63** [comment]
For posterity:  We've decided we can merge this now and PyPI it as part of CLS GA.

**Pull request merged**